lib/src/store_watcher.dart

```diff
@@ -121,9 +126,8 @@ abstract class StoreWatcherMixin implements State<dynamic> { // ignore: TYPE_ARG
     // TODO(abarth): We cancel our subscriptions in [dispose], which means we
     // shouldn't receive this callback when we're not mounted. If that's the
     // case, we should change this check into an assert that we are mounted.
-    if (!mounted)
-      return;
-    setState(() { });
+    if (!mounted) return;
```

Comment:
this and the other change below seem to be extraneous and violate the flutter style guide.
there's various other things that could be done in this file to improve its adherence to the style guide.

Reply:
We don't use dartfmt. It doesn't implement Flutter's style guide.

Reply:
I didn't know that, I restored the original formatting.
So there is no automated way to format to Flutter's style guide ?